### ISSUE TITLE
DBeaver 3.6.2 unresponsive executing long text query (MS SQL Server)

### ISSUE THREAD

**Issue by meddario** [serge-rider/dbeaver#304]
Hi,

i don't know if this is somehow related to issue #154 (which seems solved in the last version), i found that DBeaver becomes unresponsive when executing a very long query (8000+ lines) to SQL Server.
The unresponsiveness shows up when clicking/scrolling in the results table.

I attach a thread dump.
[thread-dump-3.6.2-big-query.txt](https://github.com/serge-rider/dbeaver/files/186107/thread-dump-3.6.2-big-query.txt)

OS: Ubuntu 15.10
Java: 
```
openjdk version "1.8.0_66-internal"
OpenJDK Runtime Environment (build 1.8.0_66-internal-b17)
OpenJDK 64-Bit Server VM (build 25.66-b17, mixed mode)
```
SQL Server 2014

Regards,
Dario

**Comment by serge-rider** [comment]
It is not related to #154. Seems to be a combination of different issues in DBeaver and in SWT - as a result UI freezes during execution of very long queries.
Thanks for report (thread dump was very useful).

**Comment by meddario** [comment]
Hi, sorry for the delay, i confirm that the problem is fixed in 3.6.3! Thank you very much!
